test/e2e-tests-examples.sh

```diff
@@ -60,10 +61,65 @@ check_eventlistener() {
   kubectl get eventlisteners
 }
 
-trap "cleanup" EXIT SIGINT
+ignore=( cron trigger-ref )
+
+port_forward_and_curl() {
+
+  if [[ " ${ignore[@]} " =~ " ${current_example} " ]]; then
+    info 'ignoring for port forwarding'
+    return
+  fi
+
+  info "Port forwarding to execute curl command"
+
+  kubectl port-forward service/el-${current_example}-listener 8080:8080 &
+  port_forward_pid=$!
+
+  # Wait a few seconds for forwarding to happen
```

Comment:
hmm I wonder if there is a way we can wait for port-forward to succeed instead of a sleep...but if there is nothing obvious, this is ok for now